setup.py

```diff
@@ -22,6 +22,7 @@
         'cftime',
         'f90nml',
         'joblib',
+        'ipywidgets',
```

Comment:
I had this already installed in `pip`, but I needed to run `jupyter nbextension enable --py widgetsnbextension` to make it active. Sounds like this isn't a problem for the conda environment, but maybe just something to keep in mind in case others have issues.

Reply:
Oh right. I find all this extension stuff more than a little painful. AFAICT this doesn't work in jupyter lab under the NCI `conda` environments because the relevant extensions haven't been installed. **grrr**

I installed it for travis. Tests fail otherwise. Happy to move that dependency somewhere else if there is a more appropriate location.